engine/installation/index.md

```diff
@@ -28,18 +32,36 @@ Docker is available in multiple variants:
   share containers and automate the development pipeline all from a single
   environment.
 
-  Docker CE gives you the option to run **stable** or **edge** builds.
+  Docker CE gives you the option to subscribe to the **stable** or **edge**
+  channel.
+
+  - The **Stable** channel receives new Docker version once per **quarter**. After
+    a new stable release, the previous release is supported for one month. This
+    support includes security updates and other bug fixes.
 
-  - **Stable** builds are released once per quarter.
-  - **Edge** builds are released once per month.
+  - The **Edge** channel receives a new Docker version once per **month**. This
+    release is only supported during that month, and includes security updates
+    and high priority bug fixes only. When a new Edge release becomes available,
+    you should upgrade right away. The Edge channel gives you access to new
+    features more quickly.
+    [Learn about current Edge-only features.](/engine/edge.md).
 
   For more information about Docker CE, see
   [Docker Community Edition](https://www.docker.com/community-edition/){: target="_blank" class="_" }.
 
-- **Docker Cloud** is a platform run by Docker which allows you to deploy your
-  application using multiple cloud providers such as Digital Ocean, Packet,
-  SoftLink, or to bring your own device. For more information about using Docker
-  Cloud, see [Docker Cloud](#docker-cloud).
+The following table shows what to expect from the different Docker channels.
```

Comment:
The table is great; looking at it on the page; perhaps add a heading in between? "Release schedule" / "Support cycle" / "Something else" ?

<img width="949" alt="screen shot 2017-03-22 at 17 21 08" src="https://cloud.githubusercontent.com/assets/1804568/24208542/1b86c1da-0f24-11e7-9396-804eb1b454b3.png">